interfaces/Portalicious/src/app/services/auth/strategies/msal-auth/msal-auth.app-providers.ts

```diff
@@ -80,6 +81,12 @@ function MSALGuardConfigFactory(): MsalGuardConfiguration {
   };
 }
 
+export function getOriginUrl(): string {
+  return !environment.production
+    ? window.location.origin
+    : `${window.location.origin}/${Locale.en}`;
```

Comment:
```suggestion
    : `${window.location.origin}/${environment.defaultLocale}`;
```

A bit more future-proof/flexible?